test/test.lua

```diff
@@ -0,0 +1,231 @@
+-- To run this file use luajit binary as below
```

Comment:
@abhay1722 I wonder if this test file has something that is not yet covered by the luajit2-test-suite repo?

Reply:
@abhay1722 BTW, general LuaJIT tests should go to the luajit2-test-suite repo. The test suite in this repo only exercise features specific to this (openresty's) branch of LuaJIT.

Reply:
@agentzh Thanks for pointing this out.
This is just a sample test file to check the s390x assembly code added in the repo, to check simple operations like addition,loops,functions etc. It doesn't affect the mainline code of the (openresty's) branch of LuaJIT.
This test file can be removed now. Please let me know if the PR is good to merge now.

Reply:
@abhay1722 Fair enough.